### ISSUE TITLE
[Idea] make grenades destroyable

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/gameplay#47]
Currently, grenades can't be countered. This is pretty OP, and I think some aliens, maybe not all, shall be able to neutralize those.

**Comment by slipher** [comment]
Picking up and re-throwing them would be funnier.

**Comment by bmorel** [comment]
what about technical difficulty?

**Comment by DolceTriade** [comment]
Only the granger can rethrow nades. All other aliens don't have hands.

**Comment by bmorel** [comment]
I don't think it shuold be reserved to granger. Devolving from adv goon to granger have a real cost.

**Comment by slipher** [comment]
Larger forms can pick up the grenade in their mouths. 

Also what part of the dragoon's body does the barb canonically come out of?

**Comment by Viech** [comment]
This idea, apart from injecting a perceived problem that wasn't discussed (it is not obvious that something is overpowered simply as there is no counter to it—which in the case of the grenade is also wrong as you counter it by using your superior mobility to stop humans outside your own base), is lacking details. How exactly should aliens be able to neutralize the grenade?

**Comment by bmorel** [comment]
Well, you can remove the "pretty OP" part if you want. But I thought the point of this tracker was really to discuss ideas, not necessarily ideas that already have complete specs. If I had complete specs, I'd just implement it and host it on a mod...

**Comment by bmorel** [comment]
>  How exactly should aliens be able to neutralize the grenade?

The title of this issue says "destroyable". This means giving them some health, and being able to attack them.

As for the superior mobility, I'd say you need to prove there is such a thing. Humans run fast. Sure, aliens have some trick moves to go a bit faster, but so do humans, which even have the jetpack, are relatively small and can duck.

**Comment by Viech** [comment]
The "best" defense strategy for both teams since Tremulous days has been to meet the enemy outside your base. "Best" in the sense that (1) it does work reasonably well, in particular for aliens as humans have a hard time just slipping past them, and (2) that this dynamic is arguably much more fun than the alternative where teams defend their base from inside their base. Human base camping was one of the strategies that people disliked the most about Tremulous gameplay. Aliens, luckily, coldn't quite do this, in part due to grenades. (Except for, say, Niveus, where you had to block just one door with a Tyrant.)

If it's an optimal play to have a player inside your base to deal with fire, grenades, repairs, etc., then that can be problematic and lead to stale matches. If human base attacks were too strong, then I would prefer a solution that boosts aliens ability to control the map outside their base (i.e. force humans trying to rush or fly past them into fights).

**Comment by bmorel** [comment]
How is all that related to the idea of making grenades damageable? Besides, why are you mentioning tremulous? Unvanquished's gameplay is already quite different from trem's:

* no more SD
* a BP system that people barely understand
* no mantis
* BS are smaller and can crouch (yes, this have serious impact, for example in forlorn's default A base, thye can go under stairs while adv goons or rants can not)
* human base regenerates all the time, no need for someone to take care of it
* rocket pods are the perfect camping tool

I probably miss a few more gameplay changes.

**Comment by Viech** [comment]
**I do not mind grenades being destroyable or interactable as such.** I agree that picking them up and swallowing, tossing or carrying them could be fun additions. But I disagree that this is necessary or even helpful for team balance and/or gameplay dynamics, which is something that I believe you brought up to motivate your idea. I find that:

1. It is not correct that grenades cannot be countered. You counter them by _leaving your base_ and fighting grenade carriers outside of it.
2. This possibility to counter grenades is an incentive for fun gameplay, as I tried to outline in my last post (TL;DR:  grenades incentivize aliens to fight for map rather than to camp). This speaks against implementing grenade destruction **for balancing/dynamics reasons**.
3. Tremulous is a perfectly fine example to clarify the above point. Therein, humans had a stronger incentive to enforce slow and boring gameplay than aliens, in part, as humans had a better ability and greater necessity to defend their base from within their base. Grenades and lucifer cannons, both of which we also have and with the exact same core mechanics, are an incentive not to camp but to look for hallway fights.

These points should rather have been discussed in a *problem issue* where people can discuss whether the grenade is a problematic feature or in a good shape. We are having this discussion _here_ because you chose in your opening post to (1) not discuss your idea of destroying grenades in much depth while (2) claiming that grenades cannot be countered. You can hardly expect people to take (2) as a fact and flesh out the details of (1) for you.

> Besides, why are you mentioning tremulous?

Besides, why do you mention the horrible BP system I designed and the unfun rocket pod whose gamelogic I implemented? Is it necessary to make me feel bad about things I put effort into just to counter my Tremulous comparison?

**Comment by Viech** [comment]
**Summary of hidden posts:** It is controversial whether a change to grenades is needed for balancing or gameplay reasons. There is no agreement that they are too strong or that additional means to counter them are necessary.

We can still discuss here fun ways to interact with grenades once dropped.

**Comment by Gireen** [comment]
i allowed me to edit the issue according to bmorels wish in the hidden comment, since it qualifies as a prime example of what an idea issue should not contain. 

There is no problem attached to this so the question is only: Should grenades be destroyable by aliens for the sake of it?

In my opinion No, they are small, hard to "throw", dont do that much damage and humans have to carry them across the map.

**Comment by illwieckz** [comment]
My general opinion on many suggestions that are done is that _we may not need more complexity_. Fixing gameplay by feeding a cold war where every parts just gets more and more weaponry or weaponized behaviours is maybe just making the game more complex and maybe just contributes to more elaborated camping, leaving the game with the same camping but with higher gap for newcomers.
